setup.py

```diff
@@ -24,7 +24,7 @@
       author='Yusuke Tsutsumi',
       author_email='yusuke@tsutsumi.io',
       url='http://deepmerge.readthedocs.io/en/latest/',
-      packages=find_packages(),
+      packages=find_packages(exclude=['tests*', '*.tests*']),
```

Comment:
the first "tests" seems unnecessary. With local testing, I found that the latter wildcard match captures all modules, as there is no top level "tests" directory here.

But maybe you're arguing for preemptively catching future "tests" directories?

Reply:
Yes, the latter is enough as long as the tests are a subpackage of something.